src/targets/gpu/fuse_mlir.cpp

```diff
@@ -21,6 +21,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+#include "migraphx/shape.hpp"
```

Comment:
Use angle brackets, but I dont think this header is needed as its already included by the other headers.

Reply:
I just removed it. Probably, my vscode included it automatically